e2e/e2etests/test_bitcoin_deposit_and_call_revert.go

```diff
@@ -0,0 +1,51 @@
+package e2etests
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	"github.com/zeta-chain/node/testutil/sample"
+	zetabitcoin "github.com/zeta-chain/node/zetaclient/chains/bitcoin"
+)
+
+func TestBitcoinDepositAndCallRevert(r *runner.E2ERunner, args []string) {
```

Comment:
Should be rather called `TestBitcoinDepositAndRevert` as we don't call anything on Bitcoin

Reply:
But remove the `Call` doesn't express the operation code of `deposit_and_call`. Or do you like `TestBitcoinDepositCallAndRevert`?